gbpservice/nfp/core/cfg.py

```diff
@@ -56,6 +56,8 @@ def init(args, **kwargs):
     """Initialize the configuration. """
     oslo_config.CONF.register_opts(NFP_OPTS)
     oslo_config.CONF.register_opts(es_openstack_opts, "keystone_authtoken")
+    oslo_config.CONF.set_override('use_syslog', 'True')
+    oslo_config.CONF.set_override('syslog_log_facility', 'local1')
     oslo_config.CONF(args=args, project='nfp',
```

Comment:
We removed this override in opensource. One needs to set this manually in neutron.conf.